internal/services/paloalto/next_generation_firewall_vhub_local_rulestack_resource_test.go

```diff
@@ -233,9 +274,11 @@ provider "azurerm" {
 %[1]s
 
 resource "azurerm_palo_alto_next_generation_firewall_virtual_hub_local_rulestack" "test" {
-  name                = "acctest-ngfwvh-%[2]d"
-  resource_group_name = azurerm_resource_group.test.name
-  rulestack_id        = azurerm_palo_alto_local_rulestack.test.id
+  name                 = "acctest-ngfwvh-%[2]d"
+  resource_group_name  = azurerm_resource_group.test.name
+  rulestack_id         = azurerm_palo_alto_local_rulestack.test.id
+  marketplace_offer_id = "pan_swfw_cloud_ngfw"
+  plan_id              = "panw-cngfw-payg"
```

Comment:
Can this be `panw-cloud-ngfw-payg` so we can test the update path of this property?

Reply:
No. Service team has deprecated "panw-cloud-ngfw-payg". If we set it to "panw-cloud-ngfw-payg", API call would fail with below error.

Note: All related test cases on Teamcity Daily Run are also failed with below same error.

```
Error: creating Firewall (Subscription: "xxxxx"
Resource Group Name: "acctestRG-PANGFWVN-250116020412783950"
Firewall Name: "acctest-ngfwvn-250116020412783950"): performing CreateOrUpdate: unexpected status 400 (400 Bad Request) with error: PaymentRequired: SaaS Purchase Payment Check Failed as validationResponse was {"isEligible":false,"errorMessage":"Plan 'panw-cloud-ngfw-payg' is defined as stop sell in offer 'pan_swfw_cloud_ngfw'."}
```